locale/en/foundation/case-studies.md

```diff
@@ -16,7 +16,7 @@ preferences (current and future) for front end, back end and IoT developers.
 
 * [Read the full announcement.](/en/blog/announcements/nodejs-foundation-survey/)
 * [See the infographic.](/static/documents/2016-survey-infographic.png)
-* [Read the full report.](/static/documents/2016-survey-report.pdf)
+* <a href="/static/documents/2016-survey-report.pdf" onClick="ga('send', 'event', { eventCategory: 'casestudy', eventAction: 'click', eventLabel: 'Node.js 2016 Report', eventValue: 0});">Read the full report.</a>
```

Comment:
@fhemberger thank you. My only hesitation in doing it this way is I only know how to upload them the way that I learned... If I need to upload more will i need to create a new process here?